doc/docs/setup.md

```diff
@@ -0,0 +1,41 @@
+# Setup
+
+## Installation
+
+### Install via pip:
+
+`pip3 install https://github.com/Lonami/pyndustric/archive/refs/heads/master.zip`
```

Comment:
Inline code blocks are likely not the right choice, rather, I'd recommend:

````md
```sh
pip install ....
```
````